src/main/java/com/openClassroomsProject/SafetyNetAlerts/service/impl/FireStationServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package com.openClassroomsProject.SafetyNetAlerts.service.impl;
+
+import com.openClassroomsProject.SafetyNetAlerts.model.FireStation;
+import com.openClassroomsProject.SafetyNetAlerts.repository.FireStationRepository;
+import com.openClassroomsProject.SafetyNetAlerts.service.FireStationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class FireStationServiceImpl implements FireStationService {
+
+    @Autowired
+    private FireStationRepository fireStationRepository;
```

Comment:
Should be an interface

Reply:
FirestationRepository is an interface ^^